features/autorisatie/bewoning/raadpleeg-bewoning-met-periode/autorisatie.feature

```diff
@@ -0,0 +1,183 @@
+      | type                             | BewoningMetPeriode |
+      | datumVan                         | 2022-01-01         |
+      | datumTot                         | 2023-01-01         |
+      | adresseerbaarObjectIdentificatie | 0800010000000001   |
+      Dan heeft de response 0 bewoningen
+
+    @fout-case
+    Scenario: Gemeente raadpleegt bewoning van een adresseerbaar object buiten de gemeente
+      Gegeven de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | afnemerID | gemeenteCode |
+      | 000008    | 0800         |
+      Als bewoningen wordt gezocht met de volgende parameters
+      | naam                             | waarde             |
+      | type                             | BewoningMetPeriode |
+      | datumVan                         | 2022-01-01         |
+      | datumTot                         | 2023-01-01         |
+      | adresseerbaarObjectIdentificatie | 0518010000000002   |
+      Dan heeft de response de volgende gegevens
+      | naam     | waarde                                                                                 |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3                            |
+      | title    | U bent niet geautoriseerd voor deze vraag.                                             |
+      | status   | 403                                                                                    |
+      | detail   | Je mag alleen bewoning van adresseerbare objecten binnen de eigen gemeente raadplegen. |
+      | code     | unauthorized                                                                           |
+      | instance | /haalcentraal/api/bewoning/bewoningen                                                  |
+
+
+  Regel: Een gemeente als afnemer is geautoriseerd voor het vragen van bewoning van een adresseerbaar object dat niet bestaat of nog nooit bewoond is
+
+    Scenario: Het adresseerbaar object wordt niet gevonden, waardoor de gemeente van het adresseerbaarbaar object niet gelijk of ongelijk kan zijn aan de afnemer
+      Gegeven de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | afnemerID | gemeenteCode |
+      | 000008    | 0800         |
+      Als bewoningen wordt gezocht met de volgende parameters
+      | naam                             | waarde             |
+      | type                             | BewoningMetPeriode |
+      | datumVan                         | 2010-01-01         |
+      | datumTot                         | 2010-08-17         |
+      | adresseerbaarObjectIdentificatie | 1234010000123456   |
+      Dan heeft de response 0 bewoningen
+    
+
+  Regel: Een gemeente als afnemer is geautoriseerd voor het vragen van bewoning van een object dat na gemeentelijke herindeling is komen te liggen in bevragende gemeente
+
+    Abstract Scenario: Adres is na gemeentelijke herindeling in vragende gemeente komen te liggen en <omschrijving>
```

Comment:
ik begrijp niet hoe dit scenario gemeentelijke herindeling toont. Ik denk dat in dat geval er twee adressen zijn met dezelfde identificatiecode verblijfplaats, maar verschillende gemeentecode.

aangezien er nu alleen naar de adressentabel wordt gekeken en niet naar de verblijfplaatstabel, zijn de voorbeelden met datums denk ik ook niet meer relevant.

Reply:
eeehhhhm ik keek scheel. staat er wel goed